### DISCUSSION TITLE
Persistence of OBS Plugins?

### DISCUSSION THREAD

**Discussion by marvinlehmann** [ScoopInstaller/Extras#8754] [Q&A]
How would you handle persisting plugins for OBS?

Their configs are getting persisted because they're in the config folder: `.\config\obs-studio\plugin_config\...`

But dlls, pdbs or localization files are not
`.\obs-plugins\64bit\win-capture-audio.dll`
`.\data\obs-plugins\win-capture-audio\locale`

They're next to preinstalled plugins..
I guess you need something like #8017 ?

**Comment by saturdayxiii** [comment]
I don't know why the "addition" folder is only used for Mozilla apps.  It would be very helpful for any app that uses plugins, themes, addons, brushes, scripts, etc.

**Comment by HUMORCE** [comment]
> I don't know why the "addition" folder is only used for Mozilla apps. It would be very helpful for any app that uses plugins, themes, addons, brushes, scripts, etc.

This is just temporary solution for firefox, if most manifests need this functionality then I think it should be implemented in Scoop Core rather than adding the same function to every manifest.

**Comment by goyalyashpal** [comment]
was there any conclusion on this?
